enterprise_access/apps/bffs/tests/test_context.py

```diff
@@ -1,40 +1,82 @@
 """
 Text for the BFF context
 """
+import ddt
 from django.test import RequestFactory, TestCase
+from faker import Faker
 from rest_framework.exceptions import ValidationError
 
-from enterprise_access.apps.api_client.tests.test_constants import DATE_FORMAT_ISO_8601
 from enterprise_access.apps.bffs.context import HandlerContext
 from enterprise_access.apps.core.tests.factories import UserFactory
-from enterprise_access.utils import _curr_date
 
 
+@ddt.ddt
 class TestHandlerContext(TestCase):
     def setUp(self):
         super().setUp()
         self.factory = RequestFactory()
         self.mock_user = UserFactory()
+        self.faker = Faker()
+
+        self.mock_enterprise_customer_uuid = self.faker.uuid4()
+        self.request = self.factory.get('sample/api/call')
+        self.request.user = self.mock_user
+        self.request.query_params = {
+            'enterprise_customer_uuid': self.mock_enterprise_customer_uuid
+        }
+        self.request.data = {
+            'enterprise_customer_uuid': self.mock_enterprise_customer_uuid
+        }
+        self.context = HandlerContext(self.request)
 
     def test_handler_context_init(self):
-        request = self.factory.get('sample/api/call')
-        request.user = self.mock_user
-        context = HandlerContext(request)
+        context = HandlerContext(self.request)
```

Comment:
[nit/curious] Would it make sense to rely on `self.context` here as it's already instantiated in the `setUp`?

```python
self.assertEqual(self.context.request, self.request)
# ...
```